src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/ContactDetails.cshtml

```diff
@@ -18,12 +18,68 @@
             <h1 class="govuk-heading-m">
```

Comment:
@stevelee-civica  think we probably need a back link

Reply:
already in place